### PR TITLE
Spelling corrections

### PR DESCRIPTION
**Pull request by justinormont**
Barge of spelling/grammar corrections I've noted over time. Sailing the high seas, enjoying the sun, occasionally noticing the one, the oddity floating where it aught not be.

### CODE REVIEW

File: src/Microsoft.ML.FastTree/FastTreeArguments.cs [hunk 1]
```diff
@@ -349,8 +349,8 @@ public enum OptimizationAlgorithmType { GradientDescent, AcceleratedGradientDesc
         [Argument(ArgumentType.LastOccurenceWins, HelpText = "Optimization algorithm to be used (GradientDescent, AcceleratedGradientDescent)", ShortName = "oa")]
         public OptimizationAlgorithmType OptimizationAlgorithm = OptimizationAlgorithmType.GradientDescent;
 
-        [Argument(ArgumentType.Multiple, HelpText = "Early stopping rule. (Validation set (/valid) is required.)", ShortName = "esr", NullName = "<Disable>")]
-        [TGUI(Label = "Early Stopping Rule", Description = "Early stopping rule. (Validation set (/valid) is required.)")]
```
Comment (sfilipi) [inline src/Microsoft.ML.FastTree/FastTreeArguments.cs]: might want to keep this for cmd users

Comment (justinormont) [reply src/Microsoft.ML.FastTree/FastTreeArguments.cs]: The comment was out of date; the current form is `valid=...`. The comment remains, though I removed the out of date part.

**Comment by sfilipi** [comment]
@justinormont this is approved. Are you still planning to merge it?

**Comment by TomFinley** [comment]
Hi @justinormont, just to reiterate @sfilipi's question, is this dead?

**Comment by justinormont** [comment]
If I get the time, I'll re-open; otherwise feel free to push into my branch.

**Pull request closed without merging**